src/data/list/sort.lean

```diff
@@ -281,21 +312,26 @@ theorem sorted_merge : ∀ {l l' : list α}, sorted r l → sorted r l' → sort
     { exact rel_of_sorted_cons h₂ _ bl' } }
 end
 
-theorem sorted_merge_sort : ∀ l : list α, sorted r (merge_sort l)
+theorem sorted_merge_sort : ∀ l : list α, sorted r (merge_sort r l)
 | []        := sorted_nil
 | [a]       := sorted_singleton _
 | (a::b::l) := begin
   cases e : split (a::b::l) with l₁ l₂,
   cases length_split_lt e with h₁ h₂,
   rw [merge_sort_cons_cons r e],
-  exact sorted_merge r (sorted_merge_sort l₁) (sorted_merge_sort l₂)
+  exact r.merge (sorted_merge_sort l₁) (sorted_merge_sort l₂)
```

Comment:
This fixes the current error in the build. The `r` looks a little bit weird, maybe it should be implicit?
```suggestion
  exact (sorted_merge_sort l₁).merge r (sorted_merge_sort l₂)
```